core/modules/entity/entity.class.inc

```diff
@@ -278,6 +278,24 @@ abstract class Entity extends stdClass implements EntityInterface {
     return $this->entityType();
   }
 
+  /**
+   * Returns the label of the entity.
+   *
+   * @return
```

Comment:
The return type is missing: `string|null`

Reply:
Just in case you're not aware of it: a PR can have as many commits as it takes to get done, as it will be "squashed and merged", so in the end it'll appear as one commit. ... If that's your reason for redoing.

Reply:
I've added it.